scripts/build_wheel.py

```
@@ -822,18 +822,33 @@ def symlink_remove_dst_tree(src, dst, dirs_exist_ok=True):
+                    if dst_path.stat().st_mtime >= resolved_src.stat().st_mtime:
+                        return  # destination is up to date
                     rmtree(dst_path)
-                # Use symlinks=False (default) to follow symlinks and copy actual content
-                # This ensures nested symlinks are also resolved
-                copytree(resolved_src, dst_path, symlinks=False)
+                # Shell out to cp -rL: dereferences symlinks like copytree(symlinks=False)
+                # but uses kernel-level copy primitives, which is significantly faster
+                # than Python's file-by-file copytree on network-mounted filesystems.
+                # Fall back to copytree if cp is unavailable (non-Linux or minimal containers).
+                try:
+                    run(["cp", "-rL",
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)